release-engineering/packaging.md

```diff
@@ -0,0 +1,125 @@
+# Packaging Kubernetes <!-- omit in toc -->
+
+_Author(s): Sumitran Raghunathan ([@sumitranr](https://github.com/sumitranr))_  
+_Editor(s): Stephen Augustus ([@justaugustus](https://github.com/justaugustus))_  
+_Original document: [Building debs/rpms for Kubernetes
+](https://docs.google.com/document/d/1PAN5tVJO_NMfHZmnk8mDQTwAbFHPky7JBgWJgckNjro/edit?usp=sharing)_
+
+- [Introduction](#introduction)
+- [Communication](#communication)
+- [Release Steps](#release-steps)
+  - [Permissions](#permissions)
+  - [Clone Release Repository](#clone-release-repository)
+  - [Authenticate](#authenticate)
+  - [Build the Debs & RPMs](#build-the-debs--rpms)
+    - [Notes](#notes)
+  - [Validating packages](#validating-packages)
+  - [Package verification tests](#package-verification-tests)
+
+## Introduction
+
+This guide outlines the process of building debs/rpms for Kubernetes minor and patch releases.
+
+**DISCLAIMER: This document was initially drafted for Google internal release managers and has been published here (with edits) for the sake of visibility into the Kubernetes build/packaging process. With that in mind, there are several links and processes referenced here that can only be accessed/performed by a Google employee with the requisite access to Google build systems/tooling. We will refer to these Googlers as ["Kubernetes Build Admins"][kubernetes-build-admins] across this guide.**
+
+## Communication
+
+The [Patch Release Team][patch-release-team] (patch releases) or [Branch Manager][branch-manager-handbook] (minor releases) will reach out to the [Kubernetes Build Admins][kubernetes-build-admins] via [email][kubernetes-build-admins] or `#release-managers` on [Kubernetes Slack](https://slack.k8s.io) (for more synchronous communication) requesting help to build the debs and rpms.
+
+Patch Release Team members or Branch Managers requesting debs/rpms should be sure to provide explicit details on the package name(s), version(s), and revision(s) they need built.
+
+**n.b. As much as possible, communications with the Kubernetes Build Admins should happen on public forums (email, Slack public channels), not direct messaging, except in instances where doing so would run contrary to our [security policies][security-release-process] e.g., building packages for a release which addresses an embargoed CVE.**
+
+## Release Steps
+
+In this process, we are pulling the artifacts published by a Release Manager to the GCS bucket and building debs/rpms to be published to the rapture repository. 
+
+Refer to the [Branch Manager handbook][branch-manager-build-and-release] for details on the artifacts that are built by the Branch Manager.
+
+See also the [rapture documentation][rapture-readme], which contains details on running `rapture` (Google internal packaging tool).
```

Comment:
I really dislike that this document references multiple Google-internal documents.  Each of these should have an issue open for replacement by open process, with a "(see #XXXXX for resolving this Google dependency)" parenthetical reference next to each usage in this document.